File: cassandrablog/src/test/java/com/saathratri/developer/blog/domain/BlogTestSamples.java
package com.saathratri.developer.blog.domain;

import java.util.UUID;

public class BlogTestSamples {

    public static Blog getBlogSample1() {
        return new Blog()
            .compositeId(new BlogId().category("category1").blogId(UUID.fromString("23d8dc04-a48b-45d9-a01d-4b728f0ad4aa")))
            .handle("handle1")
            .content("content1");
    }

    public static Blog getBlogSample2() {
        return new Blog()
            .compositeId(new BlogId().category("category2").blogId(UUID.fromString("ad79f240-3727-46c3-b89f-2cf6ebd74367")))
            .handle("handle1")
            .content("content1");
    }

    public static Blog getBlogRandomSampleGenerator() {
        return new Blog()
            .compositeId(new BlogId().category(UUID.randomUUID().toString()).blogId(UUID.randomUUID()))
            .handle("handle1")
            .content("content1");
    }
}

File: cassandrablog/src/main/webapp/app/entities/cassandrablog/blog/blog.test-samples.ts
import { IBlog, NewBlog } from './blog.model';

export const sampleWithRequiredData: IBlog = {
  compositeId: { category: 'sample-category-1', blogId: 'sample-blogId-1' },
};

export const sampleWithPartialData: IBlog = {
  compositeId: { category: 'sample-category-2', blogId: 'sample-blogId-2' },
  handle: 'sample-handle-2',
};

export const sampleWithFullData: IBlog = {
  compositeId: { category: 'sample-category-3', blogId: 'sample-blogId-3' },
  handle: 'sample-handle-3',
  content: 'sample-content-3',
};

export const sampleWithNewData: NewBlog = {
  compositeId: { category: 'sample-category-4', blogId: 'sample-blogId-4' },
};

Object.freeze(sampleWithNewData);
Object.freeze(sampleWithRequiredData);
Object.freeze(sampleWithPartialData);
Object.freeze(sampleWithFullData);
